components/public-api/gitpod/experimental/v1/user.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package gitpod.experimental.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/public-api/experimental/v1";
+
+import "google/protobuf/timestamp.proto";
+
+message User {
+    // id is a UUID of the user
+    string id = 1;
+
+    string name = 2;
+
+    string avatar_url = 3;
+
+    // created_at is the creation time
+    google.protobuf.Timestamp created_at = 5;
+}
+
+message PublicSSHKey {
+    // id is a UUID of the SSH key
+    string id = 1;
+
+    // name is the name of the SSH key
+    string name = 2;
+
+    // key is the public SSH key
+    string key = 3;
+
+    // created_at is the creation time
+    google.protobuf.Timestamp created_at = 4;
+}
+
+service UserService {
+    // Lists the public SSH keys.
+    rpc GetUser(GetUserRequest) returns (GetUserResponse) {};
+
+
+    // Lists the public SSH keys.
+    rpc ListPublicSSHKeys(ListPublicSSHKeysRequest) returns (ListPublicSSHKeysResponse) {};
```

Comment:
If there's a Public SSH Key, where is the Private one?

Would it make sense to make it `ListSSHKeys` and the `SSHKey` object contains the property `public` or `private` depending on what we need?

Reply:
Why would you want to upload your private key? only the public key needs to be shared

Reply:
It's not about whether I would upload it. It's what a user will ask when they see `Public` in the method name. Whenever polar keywords exist (Public / Private, Running/Not Running) it raises a question about what it actually means.

Given we only every ask for Public API keys, it feels safe to drop the `Public` from the method name to reduce the mental complexity for a user. The SSH Key can still have a property which is `public_key` which makes it further more explicit.